initializr-generator-spring/src/main/java/io/spring/initializr/generator/spring/documentation/RequestedDependenciesHelpDocumentCustomizer.java

```diff
@@ -65,8 +67,11 @@ public void customize(HelpDocument document) {
 			}
 		});
 
-		modifications.ifVersionChanged((oldVersion, newVersion) -> document.addSection(
-				writer -> writer.write(String.format("OLD: %s --> NEW:%s", oldVersion, newVersion))));
+		// TODO how to handle something on custom desc and custom diff?
```

Comment:
Ignore this comment please. To handle something custom we would either pass in the typed custom diff object or use the generic property change method.